src/containers/CreateDoc/documentTypes.ts

```diff
@@ -0,0 +1,6 @@
+export default [
```

Comment:
I assume this is a config file to keep the hard coded data outside the actual code.

I like it. But I would rather give it a default name like "config", so we create a pattern where the hardcoded data is in a config file, by the side of the actual file of th ecomponent.